source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -175,23 +176,29 @@ Wallets
 |mw-gen2| for Android
 ---------------------
 
-    March 20, 2024
+    June 26, 2024
+
+    Version 1.6.0 removed shielding. You can still enable and see your shielded balance and history, but to unshield the funds |cryptox| must be used.
```

Comment:
```suggestion
    Version 1.6.0 removes shielding. You can still see your shielded balance and history, but to unshield the funds |cryptox| must be used.
```
I'm not quite sure what it means to "enable" your shielded balance.

Reply:
In order to see the shielded balance, you have to go to the account settings and enable it from there.